docker-multinode/common.sh

```diff
@@ -273,7 +273,7 @@ kube::multinode::turndown(){
     kube::log::status "Killing docker bootstrap..."
 
     # Kill the bootstrap docker daemon and it's containers
-    docker -H ${BOOTSTRAP_DOCKER_SOCK} rm -f $(docker -H ${BOOTSTRAP_DOCKER_SOCK} ps -q) >/dev/null 2>/dev/null
+    docker -H ${BOOTSTRAP_DOCKER_SOCK} rm -f $(docker -H ${BOOTSTRAP_DOCKER_SOCK} ps -a -q) >/dev/null 2>/dev/null
```

Comment:
What about the kubernetes containers a few lines below? 

BTW: it is not possible to see logs after running turndown.sh. But I think this is ok. (I assume everyone will troubleshoot while cluster is running)